packages/mobile/src/transactions/send.ts

```diff
@@ -114,6 +115,15 @@ export function* sendTransactionPromises(
   const fornoMode: boolean = yield select(fornoSelector)
   let gasPrice: BigNumber | undefined
 
+  // If stableToken is prefered to pay fee, use it unless its balance is Zero,
+  // in that case use CELO to pay fee.
+  // TODO: Make it transparent for the user.
+  // TODO: Check for balance should be more than fee instead of zero.
+  const feeCurrency =
+    preferredFeeCurrency === CURRENCY_ENUM.DOLLAR && stableTokenBalance.isGreaterThan(0)
```

Comment:
This was working like this before and I was worried about changing it in case someone only has CELO and needs to pay for gas for something, but I would understand not wanting to have it anymore.

Reply:
Doesn't this create a potential discrepancy between what we show users and what we actually do? For example, if someone has both cUSD and CELO, we will always show them the fee in CELO but then will always deduct their fee from their cUSD balance.

If my understanding is correct, I think we need to get rid of this. It looks like you've set the default currency = cUSD for the param in the relevant function calls but I'd definitely test it out to be safe.

Reply:
Gotcha, I totally misread this the first time. Sounds good!